### PR TITLE
SSBFFA: Refactor using lib/fs.js

### PR DESCRIPTION
**Pull request by DeathlyPlays**

### CODE REVIEW

**Comment by DeathlyPlays** [comment]
^ taken care of

**Comment by Volco** [comment]
All works? no errors malformed json?

**Comment by Volco** [comment]
also why did you get rid of the if for noWrite? that makes it useless if the file doesn't exist now or crashes for whatever reason

**Comment by DeathlyPlays** [comment]
👍 and I removed it because it for some reason caused a bug with not actually making the file like I do now

**Comment by Volco** [comment]
then why are the other 2 "noWrite" vars still there? remove them in that case please

**Comment by Volco** [comment]
@DeathlyPlays since the last two fs things you did caused crashes did you test this WITH JSON DATA?

**Comment by DeathlyPlays** [comment]
Test it yourself @Volco

File: wavelength-plugins/ssb.js [hunk 1]
```diff
@@ -1,9 +1,8 @@
 "use strict";
 
 global.TeamValidator = require("../sim/team-validator");
-let fs = require("fs");
+let FS = require("../lib/fs");
```
Comment (HoeenCoder) [inline wavelength-plugins/ssb.js]: const

**Comment by DeathlyPlays** [comment]
I did test

**Comment by DeathlyPlays** [comment]
Been awhile though

**Comment by DeathlyPlays** [comment]
Knew this didn't look right

File: wavelength-plugins/ssb.js [hunk 2]
```diff
@@ -578,40 +577,32 @@ class SSB {
...
-	fs.writeFile("config/ssb.json", "{}", function (err) {
-		if (err) {
-			console.error(`Error while loading SSBFFA: ${err}`);
-			ssbWrite = false;
-		} else {
-			console.log("config/ssb.json not found, creating a new one...");
-		}
-	});
-	noRead = true;
+	ssbData = FS("config/ssb.json").readIfExistsSync();
```
Comment (HoeenCoder) [inline wavelength-plugins/ssb.js]: There is no way you tested this.

Your trying to write to a `const` variable, that's a syntax error on server startup.

**Pull request closed without merging**